knative-operator/pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -95,6 +96,45 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 			return err
 		}
 	}
+
+	// Watch for kn ConsoleCLIDownload resources
+	knToRequestsFunc := handler.ToRequestsFunc(func(obj handler.MapObject) []reconcile.Request {
+		annotations := obj.Meta.GetAnnotations()
```

Comment:
This seems to be a copy of the function used above in kourier. Should we unify the used constants and define the function ony once?

Reply:
I was inclined to do this but waited for feedback before changing kourier bits.

Reply:
We can also do this in a followup.